### ISSUE TITLE
Scan directory offline with a report.

### ISSUE THREAD

**Issue by MariasStory** [saferwall/saferwall#335]
Dear authors, you have done a very good job with this interesting solution.
I would like to set it up for scanning directories with hundreds of thousands of files.
It would be nice to have the files scanned offline and have a report generated.

Is it possible to implement such a feature?

Thank you.

**Comment by LordNoteworthy** [comment]
Hello @MariasStory thank you !

1. By offline you mean, locally on your own networks ?
2. What kind of reports ? a JSON document with all the results or something visual like HTLM ?

Cheers.

**Comment by MariasStory** [comment]
Hi @LordNoteworthy,

1. Local scan. Preferably, with docker options "--rm -it --network none".
2. JSON would be sufficient. Although, integrating it in HTML report would also be a good addition.

My basic idea is to scan a forensic copy with multiple AVs locally.
You already build the needed docker images. I guess that it wouldn't be hard to implement such functionality.

Thank you.

**Comment by LordNoteworthy** [comment]
Hi @MariasStory 

Yes, this should be easily doable, I am indeed working on that, this should be finish in a week.

Are you using vagrant or k8s ?

Cheers.

**Comment by MariasStory** [comment]
Hi @LordNoteworthy, thanks for support.
I did run some of your docker av implementations, works really cool.
On another hand, I did not install the solution, as I am a bit confused about the right way to implement it.
I have an Ubuntu server and (at the moment) don't want to have additional virtualization or distribution.
I am wandering if Kind will do?
Basically, I would like to run the AV on the folder in parallel (there is enough resources). Even a basic docker scripting would do it.
I am sorry, I am not an expert in kubernetes.
